docs/README.md

```diff
@@ -5,4 +5,17 @@
 - HTML: `make html`
 - PDF: `make latexpdf`
 
-TODO: Explain better
\ No newline at end of file
+## Build Guide
+- make a venv: `python3 -m venv virtualenv`
+- active virtual environment: `source virtualenv/bin/activate`
+- Install dependencies: `pip install -r requirements.txt && pip install tgcrypto`
+- Install pyrogram from source: `python setup.py install --user`
+- Change Directory to `docs` and install depenedencies: `cd docs && pip install -r requirements.txt`
+- Run releases inside `scripts` directory: `cd scripts && python releases.py`
+- Change back to project root and run `cd .. && cd .. && python setup.py generate --docs`
+- Finally go to `docs` directory and run `make html`: `cd docs && make html`
```

Comment:
all of this can be simplified. also, I've added a live-html build (with `make lhtml`) a while ago, it could be mentioned here.

Reply:
Oh im currently using that i found in `Makefile`. will push new changes tomorrow, no worries

Reply:
I have resolved it by editing the last line. works well!